opensfm/src/geometry/python/pybind.cc

```diff
@@ -251,7 +260,7 @@ PYBIND11_MODULE(pygeometry, m) {
            py::return_value_policy::copy)
       .def("__deepcopy__", [](const Camera& c, const py::dict& d) { return c; },
            py::return_value_policy::copy);
-  m.def("compute_camera_mapping", ComputeCameraMapping);
+  m.def("compute_camera_mapping", RunComputeCameraMapping);
```

Comment:
There seem to be a nice syntax we can use to do this without declaring the extra fuction
https://pybind11.readthedocs.io/en/stable/advanced/misc.html#global-interpreter-lock-gil
```suggestion
  m.def("compute_camera_mapping", RunComputeCameraMapping, py::call_guard<py::gil_scoped_release>());
```

Reply:
Done.